In [1]:
delta = 0.012
rho = 1.225
s = 0.05
A = 0.503
omega = 300
r = 0.4
U_tip = 120
k = 0.1
v_0 = 4.03
d_0 = 0.6

P_0 = (delta / 8) * rho * s * A * (omega**3) * (r ** 3)

mu_1 = P_0
mu_2 = P_0 *(3 / (U_tip ** 2))
mu_3 = P_0 * v_0
mu_4 = (1/2) * d_0 * rho * s * A

print("mu_1=", mu_1)
print("mu_2=", mu_2)
print("mu_3=", mu_3)
print("mu_4=", mu_4)

mu_1= 79.85628000000003
mu_2= 0.016636725000000005
mu_3= 321.82080840000015
mu_4= 0.009242624999999999


$$ E(u,d) = ​\mu_1\frac{d}{v}+\mu_2dv+\mu_3\frac{d}{v^2}+\mu_4dv^2 $$
$$= 79.8563\frac{d}{v}+0.0166dv+321.8208\frac{d}{v^2}+0.0092dv^2$$